app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="sp_10">10sp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.sp_10` appears to be unused
 Unused resources make applications larger and slow down builds.